src/domain/valueObject/accountId.go

```diff
@@ -26,10 +27,10 @@ func NewAccountIdPanic(value interface{}) AccountId {
 	return accId
 }
 
-func (id AccountId) Get() uint64 {
-	return uint64(id)
+func (vo AccountId) Get() uint64 {
```

Comment:
Also add on the task to replace Get() with Read() as we're already doing on newer code.

Reply:
Done.